libs/ng-mocks/src/lib/mock-service/mock-service.spec.ts

```diff
@@ -50,7 +50,7 @@ class ChildClass extends ParentClass {
   }
 }
 
-class GetterSetterMethodHuetod {
+class GetterSetterMethod {
```

Comment:
I'm not quite sure what this was doing, but afaict `Huetod` doesn't add any particular value...

Reply:
please discard.

Reply:
Out of curiosity, what is `Huetod` here?

Reply:
that's an intentional riddle :)